plugin/group/quorum.go

```diff
@@ -50,6 +50,7 @@ func (q *quorum) Stop() {
 func (q *quorum) Run() {
 	ticker := time.NewTicker(q.pollInterval)
 
+	q.converge()
```

Comment:
This is scratching an itch, the result is instances are created immediately when first `watch`ing a group.  For the script, it means a shorter `sleep`.